person_capture/gui_app.py

```
@@ -10722,6 +10722,190 @@ def _save_hdr_archive_crop(
+            backed_up_ref = False
+            backed_up_clean = False
+            restored_ref = False
+            restored_clean = False
+            publish_succeeded = False
+            try:
+                for bak in (ref_backup, clean_backup):
+                    try:
+                        if os.path.exists(bak):
+                            os.remove(bak)
+                    except Exception:
+                        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)